t/19_console_redirection.t

```diff
@@ -88,39 +82,37 @@ subtest '[connect_target_to_serial] Scenario: console already redirected' => sub
 
 subtest '[disconnect_target_from_serial]' => sub {
     my $redirect = Test::MockModule->new('sles4sap::console_redirection', no_auto => 1);
-    my $wait_serial_done = 0;    # Flag that code entered while loop
-    $redirect->redefine(record_info => sub { return 1; });
-    $redirect->redefine(wait_serial => sub { $wait_serial_done = 1; return ':~'; });
-    $redirect->redefine(enter_cmd => sub { return 1; });
-    $redirect->redefine(check_serial_redirection => sub { return $wait_serial_done; });
-    $redirect->redefine(set_serial_term_prompt => sub { return 1; });
+    my $redirection_status = 1;
+    $redirect->redefine(record_info => sub { return; });
```

Comment:
I like to redefine `record_info` with something like

```
$redirect->redefine(record_info => sub { note(join(' ', 'RECORD_INFO -->', @_)); });

```

Reply:
I usually redefine it like that if there is some useful information to show. In this case UT is checking if OpenQA parameter was set and as you can see the `record_info` does not show anything relevant there: 
```# Subtest: [disconnect_target_from_serial]
    # RECORD_INFO --> Redirect OFF Serial redirection closed. Console set to: 
    ok 1 - Restore "AUTOINST_URL_HOSTNAME" to original value
    1..1
```